requirements/developer.txt

```diff
@@ -11,3 +11,6 @@ protobuf
 grpcio-tools
 transformers==2.10.0
```

Comment:
Please update this as well

Reply:
@chauhang This is being updated as part of https://github.com/pytorch/serve/pull/995/files